src/e2e/java/teammates/e2e/util/ReportAnalysis.java

```diff
@@ -0,0 +1,62 @@
+package teammates.e2e.util;
+
+/**
+ * Report analysis class for L&P tests.
+ */
+public class ReportAnalysis {
+
+    private double meanResTime;
+    private double pct1ResTime; // 90th percentile
+    private double throughput;
+    private int errorCount;
+    private int sampleCount;
+    private String analysisFeedback = "";
+    private String testName;
+
+    /**
+     * Generate a feedback message based on test analysis with the provided threshold.
+     */
+    public void generateAnalysisFeedback(double errorRateLimit, double meanResTimeLimit) {
+        checkErrorLimit(errorRateLimit);
+        checkMeanResTimeLimit(meanResTimeLimit);
+        if ("".equals(analysisFeedback)) {
+            System.out.println("You have successfully passed the default profiling threshold for " + testName);
+        } else {
+            throw new AssertionError(analysisFeedback);
+        }
+    }
+
+    /**
+     * Generate an analysis of the test result.
+     */
+    public void generateCompleteAnalysis() {
+        System.out.print(formatAnalysis());
```

Comment:
`log.info()` instead?

Also, do we need `formatAnalysis()`?

Reply:
I chose `System.out.println` as it will output in the same format as Summary from `Summariser`

Reply:
Hmm fair point.
Actually, can you try out `Reporter.log()`?